tests/utils/streams/mixins.py

```diff
@@ -77,9 +77,9 @@ def revocation_callback(partitions: Sequence[Partition]) -> None:
             with pytest.raises(ConsumerError):
                 consumer.seek({Partition(topic, 1): 0})
 
-            consumer.pause([Partition(topic, 0)])
+            # consumer.pause([Partition(topic, 0)])
 
-            consumer.resume([Partition(topic, 0)])
+            # consumer.resume([Partition(topic, 0)])
```

Comment:
These are not implemented in the abstract class yet, I will bring these back in a near future change because they are the last big thing required for the synchronized consumer (if I recall my previous plans correctly.)